src/app/shared/directives/tooltip.directive.ts

```diff
@@ -0,0 +1,63 @@
+import { ApplicationRef, ComponentRef, createComponent, Directive, EventEmitter, HostListener, Input, OnDestroy, Output, TemplateRef } from '@angular/core'
+import { TooltipComponent } from '../components/tooltip/tooltip.component'
+
+export interface TooltipMetadata {
+  xPosition: number
+  width?: number
+}
+
+@Directive({
+  selector: '[sofieTooltip]',
+})
+export class TooltipDirective implements OnDestroy {
+  @Input() public sofieTooltip: TemplateRef<unknown>
```

Comment:
We could do that, but that will mean that instead of writing:
![image](https://github.com/tv2/ng-sofie/assets/23049647/dcef2840-4d51-44cc-a069-c0176094a54e)
We would have to write:
![image](https://github.com/tv2/ng-sofie/assets/23049647/6bc0384a-4c9b-432a-961d-2b0dff357c04)
And I think it make sense that when we declare that an element has a tooltip, we immediatly give it the template used for the tooltip.

Reply:
I think having the `Template` suffix is nice to have. If I were to create a component programmatically, I would from the attribute know that I just can't pass in the component that I created.

Reply:
I have updated to have the suffix.